### ISSUE TITLE
`import/no-unresolved` has false positives for search query

### ISSUE THREAD

**Issue by SimenB** [benmosher/eslint-plugin-import#1739]
![image](https://user-images.githubusercontent.com/1404810/80574033-5db58900-8a01-11ea-8576-bb04267c922c.png)

Queries are supported: https://nodejs.org/api/esm.html#esm_url_based_paths

Using `eslint@6.8.0` and `eslint-plugin-import@2.20.2

**Comment by ljharb** [comment]
Gross, but you're right, they are :-)

**Comment by cansin** [comment]
Is there a workaround that anyone stumbled upon until this gets fixed?

**Comment by Airkro** [comment]
Maybe we need something like `eslint-import-resolver-query` ?

**Comment by ljharb** [comment]
I think the node resolver needs to handle them by default.

**Comment by AliLozano** [comment]
Another example: This library uses the query in imports for dynamically import different things.: https://github.com/jpkleemans/vite-svg-loader

```
import iconRaw from './my-icon.svg?raw'
import IconComponent from './my-icon.svg?component'
import iconUrl from './my-icon.svg?url'
```

**Comment by JounQin** [comment]
In `eslint-import-resolver-typescript`, this is implemented without relied on `resolve` module.

https://github.com/import-js/eslint-import-resolver-typescript/blob/master/src/index.ts#L223-L230

**Comment by qlaffont** [comment]
Any update on this ? :)

**Comment by ljharb** [comment]
@qlaffont like everywhere on github, if there's updates, they'd be posted on the issue.

**Comment by Mikilll94** [comment]
This issue is still present. Any update on this?
